app/models/miq_server/worker_management/kubernetes.rb

```diff
@@ -50,6 +46,9 @@ def cleanup_orphaned_worker_rows
     end
   end
 
+  def cleanup_orphaned_workers
+  end
```

Comment:
Can you add a comment in here just to describe why this is empty?

Reply:
Same comment for the other "empty" methods.

Reply:
It just isn't implemented yet, I'm testing with Process first